src/ui/QuerySummary/QuerySummary.ts

```diff
@@ -45,19 +54,59 @@ export class QuerySummary extends Component {
    */
   static options: IQuerySummaryOptions = {
     /**
-     * Specifies whether to display the search tips to the end user when there are no search results.
+     * Specifies whether to hide the information about the currently displayed range of results and only display the
+     * search tips instead.
+     *
+     * Default value is `false`.
+     */
+    onlyDisplaySearchTips: ComponentOptions.buildBooleanOption({ defaultValue: false }),
+
+    /**
+     * Specifies whether to display the a message to the end user when there are no search results.
      *
      * Default value is `true`.
      */
-    enableSearchTips: ComponentOptions.buildBooleanOption({ defaultValue: true }),
```

Comment:
This variable still exists in the code, it just have been moved for consistency with the layout of the options.